app/src/pages/inside/dashboardItemPage/widgetsGrid/widget/widgetHeader/widgetHeader.jsx

```diff
@@ -63,6 +63,10 @@ const widgetModeMessages = defineMessages({
     id: 'WidgetHeader.widgetMode.pieChartMode',
     defaultMessage: 'Pie chart view',
   },
+  pie: {
```

Comment:
Just change field name at [this](https://github.com/reportportal/service-ui/pull/1680/files#diff-7ac12b43f6c4c64e8fd39c37433b89ccR62) line to `pie`.